### DISCUSSION TITLE
An unexpected error occurred, when uploading new logo image

### DISCUSSION THREAD

**Discussion by AliShaatani** [requarks/wiki#4840] [Error / Bug Report]
when pressing the logo image switcher, I get this error 

![screencapture-help-plutu-demo-ly-a-general-2021-12-29-12_31_14](https://user-images.githubusercontent.com/45512144/147653223-6f8a6f06-e386-456e-ae5c-8a8124439fd4.png)

and when I uploaded new logo and I want to apply it , I get the same error again 

![screencapture-help-plutu-demo-ly-a-general-2021-12-29-12_31_42](https://user-images.githubusercontent.com/45512144/147653226-fb1df16a-8470-4636-830c-f3b722f2d84b.png)

**Comment by Maurizio-Bianchi** [comment]
I got the same error and I'm in the same situation. Any clue?

My configuration:
OS: AlmaLinux 8.9 and NodeJS v14.21.3
Wiki.js version: 2.5.301
Database engine: MySQL Percona 5.7

**Comment by smeyernyc** [comment]
Same issue.
JS 20.11.1
wiki 2.5.303
mysql 5.7.44
